src/BlazorUI/Bit.BlazorUI/Components/Buttons/BitSplitButton/BitSplitButton.Fluent.scss

```diff
@@ -1,7 +1,7 @@
 ﻿@import "../../../Styles/Fluent/colors.fluent.scss", "../../../Styles/functions.scss";
 
-.bit-splt-btn {
-    &.bit-splt-btn-fluent {
+.bit-spl {
```

Comment:
it should be `spb` not `spl`

Reply:
The abbreviation `spb` has already been used in the `BitSpinButton` component and we cannot reuse it.